src/lib/features/playback-controls/ui/scrubbing-bar.svelte

```diff
@@ -48,6 +55,11 @@
     --range: calc(var(--max) - var(--min));
     --ratio: calc((var(--value) - var(--min)) / var(--range));
     --sx: calc(0.5 * 18px + var(--ratio) * (100% - 18px));
+    --played-part: linear-gradient(#818cf8, #818cf8) 0 / var(--sx) 100% no-repeat;
+    --underlying-layer: #64748b;
```

Comment:
Question: wouldn't TailwindCSS be better for consistency?

Reply:
It certainly would, but in this particular case there's just no decent way to incorporate it. Not that I see, anyway.

Reply:
I see you use auto-generated code. Then since we won't have to manually edit these styles, I think it's fine to have plain CSS.